libraries/tools/kotlin-stdlib-gen/src/templates/SpecialJVM.kt

```diff
@@ -373,7 +373,22 @@ object CommonArrays : TemplateGroupBase() {
         include(ArraysOfPrimitives, PrimitiveType.defaultPrimitives)
         include(InvariantArraysOfObjects)
     } builder {
-        doc { "Returns new array which is a copy of the original array, resized to the given [newSize]." }
+        doc {
+            """
+            Returns new array which is a copy of the original array, resized to the given [newSize], truncating or padding
+            with nulls if necessary. For all indices that are valid in both the original array and the copy, the two arrays
```

Comment:
Primitive arrays are padded with the default values of the respective primitives rather than `null`.

Perhaps this `doc` block should be only for `InvariantArraysOfObjects`, and `ArraysOfPrimitives` should get its own version.

Reply:
Should I commit the sources from templates too? I thought that the build process would do it, so I didn't commit the files

Reply:
Yes, the generated sources are in vcs, so you need to commit them.